use_cases/eluc/data/data.py

```diff
@@ -0,0 +1,196 @@
+import warnings
+import os
+import xarray as xr
+import regionmask
+import pandas as pd
+
+from datasets import load_dataset, Dataset
+
+from unileaf_util.framework.transformers.data_encoder import DataEncoder
```

Comment:
This library is not publicly available. You'll have to bing in your own version in this repo.

Reply:
Forgot this was here because prescription isn't available yet. I will work on this in a separate PR.